src/components/organisms/Sider/Sider.styled.tsx

```diff
@@ -84,16 +84,18 @@ export const StyledSiderLink = styled(SiderLink)`
   }
 `;
 
-export const StyledOtherItem = styled.div`
+export const StyledOtherItem = styled.div<{
+  $size?: number;
+}>`
   display: flex;
   justify-content: center;
   align-items: center;
 
   transition: 0.3s;
 
   svg {
-    width: 20px;
-    height: 20px;
+    width: ${({$size}) => $size}px;
+    width: ${({$size}) => $size}px;
```

Comment:
1. `width`/`height`, not `width`/`width`
2. Why is that needed? Maybe `1em` will be better?
3. `$size` may be `undefined`

Reply:
1. Agree.
2. To force `width` and `height` of icons, we kinda need it :)
3. `$size` is 20 by default.

Reply:
2. I meant basing on font size. It's common way for icons.
```
font-size: 20px;
width: 1em;
height: 1em
```

3. Looks like it's not - `$size?: number;`